tests/vm_test.go

```diff
@@ -519,7 +519,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			),
 		)
 
-		It("[test_id:6869][QUARANTINE]should report an error status when image pull error occurs", decorators.Conformance, decorators.Quarantine, func() {
```

Comment:
Please do not remove `[QUARANTINE]` yet. We can do it once we are sure the test is stable again.

Reply:
Applied the suggested change.